content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -160,52 +195,65 @@ plots:
 ![](/img/plots_cm_get_started_show.svg)
 
-Let's save this iteration so we can compare it later:
+Alternatively:
+
+- If you're using VSCode's [DVC extension] for your project, you can use the
+  [Plots Dashboard] inside your IDE.
+- If you're using [Studio] for collaboration on your ML project, use the "[Show
+  plots][studio display plots]" button in the project experiment screen. From
+  [Studio], all visualization, comparisons and trends can be easily [shared with
+  others][studio share a project] as well!
+
+Let's save this iteration into Git:
 
 ```cli
 $ git add .gitignore dvc.yaml dvc.lock eval
 $ git commit -a -m "Create evaluation stage"
 ```
 
-Later we will see how to
-[compare and visualize different pipeline iterations](#comparing-iterations).
-For now, let's see how to capture another important piece of information which
-will be useful for comparison: parameters.
+Now, let's see how to capture another critical piece of information for pipeline
+execution: parameters. Tracking parameters along with your pipelines' data is
+essential to making them fully reproducible and for future comparisons and
+analysis to be complete.
 
-[dvclive]: /doc/dvclive
-[`roc_curve`]:
-  https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html
-[`confusion_matrix`]:
-  https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html
 [plots files]: /doc/user-guide/project-structure/dvcyaml-files#plots
-[plots dashboard]:
-  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md
 [dvc extension]:
   https://marketplace.visualstudio.com/items?itemName=Iterative.dvc
+[plots dashboard]:
+  https://github.com/iterative/vscode-dvc/blob/main/extension/resources/walkthrough/plots.md
+[Studio]: /doc/studio
+[studio display plots]:
+  /doc/studio/user-guide/projects-and-experiments/visualize-and-compare#display-plots-and-images
+[studio share a project]:
+  /doc/studio/user-guide/projects-and-experiments/share-a-project#share-a-project
 
 ## Defining stage parameters
 
-It's pretty common for data science pipelines to include configuration files
-that define adjustable parameters to train a model, do pre-processing, etc. DVC
-provides a mechanism for stages to depend on the values of specific sections of
-such a config file (YAML, JSON, TOML, and Python formats are supported).
+It's common for data science pipelines to include configuration files that
+define adjustable parameters to control code execution and inject external
+inputs - whether it's model training, data processing or any other processing
+stage. DVC provides a mechanism for stages to depend on the values of specific
+sections of such a config file (YAML, JSON, TOML, and Python formats are
+supported).
 
-Luckily, we should already have a stage with
-[parameters](/doc/command-reference/params) in `dvc.yaml`:
+Luckily, the `featurize` stage already has
```

Comment:
same here, arguably not a needed change that makes it harder to review things. I know it's very hard not to do this (I'm also guilty), but we were trying a lot to split such things and/or avoid them

Reply:
Motivation behind the edit: mentioning that we're showing the featurize stage specifically before actually showing it.
This goes back to the rationale behind a lot of the changes here, I can see how they may be confusing (some of them look more substantial and other like random small edits). But they do the same - allow the guide to be read linearly and reducing mental load. never showing a command, file, or piece of configuration without FIRST explaining what it is you're looking at. Again, we have a lot of data to support that this is an issue. the guide needs to "lower it's expectation bar" to the user's understanding the context - not expect users to expand things or scroll back and forth to orient and understand what they're looking at

Reply:
yep, I think I understand the intention, it's fine. Just looked minor to me. It's subsjective, resolving thits.